test/ufuzz.js

```diff
@@ -348,7 +348,7 @@ function createSwitchParts(recurmax, n, canThrow, canBreak, canContinue, cannotR
     var s = '';
     while (n-- > 0) {
         hadDefault = n > 0; // disables weird `default` clauses until handling stabilizes
-        if (hadDefault || rng(4) > 0) {
+        if (hadDefault || rng(10) > 0) {
```

Comment:
Considering the number of errors the fuzzer found in `switch` `default:` handling, I think 25% odds of generating a `default:` clause is better. Perhaps you could change it to 20% odds if you think it's too high.